facdb/geocode/function1B.py

```diff
@@ -84,7 +84,7 @@ def parser(self, geo):
             ),
             geo_nta2010=geo.get("Neighborhood Tabulation Area (NTA)", None),
             geo_nta2020=geo.get("2020 Neighborhood Tabulation Area (NTA)", None),
-            geo_council=geo.get("City Council District", None),
+            # geo_council=geo.get("City Council District", None),
```

Comment:
Just a little unsure of the downstream machinery here - what's the affect of this?

Reply:
By commenting it out we're not assigning the City Council District from GeoSupport, so all geo_council values will be blank when it gets to the step where values are assigned via a spatial join, focing all values to be assigned spatially. 
If that doesn't answer your question, pop on over.